src/Support/Contracts/ExceptionHandler.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+
+namespace Nuwave\Lighthouse\Support\Contracts;
+
+
+interface ExceptionHandler
+{
+    public function handler(array $errors) : array;
+}
```

Comment:
newline

Reply:
@spawnia Haha sorry! :D I always always forget that...